server/tests/api/users/users.ts

```diff
@@ -273,6 +273,12 @@ describe('Test users', function () {
 
       expect(userMe.specialPlaylists).to.have.lengthOf(1)
       expect(userMe.specialPlaylists[0].type).to.equal(VideoPlaylistType.WATCH_LATER)
+
+      // Check stats are included with withStats
+      expect(userGet.videosCount).to.be.a('number')
```

Comment:
Checking field types is not enough: upload videos, add comments, abuses, update abuse states and check stats are correct